### ISSUE TITLE
Missing colors from Ansible playbook recap

### ISSUE THREAD

**Issue by huyz** [xavierog/moulti#19]
I could have sworn that the Ansible playbook recap used to show colors

But now they're monochrome:
![screenshot 2024-09-23T032826Z](https://github.com/user-attachments/assets/10698a14-98b5-46f1-bcd7-faba1d3a8a9a)

Am I misremembering?

**Comment by xavierog** [comment]
You remember correctly. Ansible generates the`PLAY RECAP` and diff (`-D`) parts using a function that relies on `sys.stdout.isatty()` to determine whether it should actually produce colors. This function obeys the environment variable `ANSIBLE_FORCE_COLOR` though. Therefore, we can solve that:
```bash
export ANSIBLE_FORCE_COLOR=yes
moulti run -- ansible-playbook -D playbook.yaml
```

![moulti-ansible-colors-1](https://github.com/user-attachments/assets/8f254011-ba89-4a12-85d5-23686d59b6a1)

I have pushed commit https://github.com/xavierog/moulti/commit/b96fae6a43198d4190b76a760bdb1ca913e227db to the devel branch so that ANSIBLE_FORCE_COLOR=yes is set out of the box.

**Comment by xavierog** [comment]
This bugfix is now available as part of [Moulti 1.21.0](https://github.com/xavierog/moulti/releases/tag/v1.21.0).
